RoboticonColony/Assets/Code/RoboticonCustomisation.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+
+sealed public class RoboticonCustomisation
+{
+    public string Name { get; private set; }
+
+    /// <summary>
+    /// Prerequisite customisations that a roboticon must have in order for this one to be applied
+    /// </summary>
+    public List<RoboticonCustomisation> Prerequisites { get; private set; }
```

Comment:
That would be sufficient at the moment but makes our code harder to maintain and extend in the future as it would make customisations limited in that sense.
The code is working and not difficult at all so I dont see any harm in leaving it like this, a list isnt exactly hard to use